versions.props

```diff
@@ -42,6 +42,7 @@ com.palantir.human-readable-types:* = 1.6.0
 org.derive4j:* = 1.1.1
 com.google.auto:auto-common = 1.2.1
 com.google.testing.compile:compile-testing = 0.19
+org.wildfly.common:wildfly-common = 1.6.0.Final
```

Comment:
I think you'll need to add `runtimeOnly 'org.wildfly.common:wildfly-common'` to the module which brings this in (likely conjure-java-undertow-lib) otherwise the version bump won't be applied to dependent projects.

Reply:
Yep, you're right that this is needed, added!

In the case I was looking at, wildfly gets introduced through:

```
com.palantir.conjure.java:conjure-java-undertow-runtime
    +--- conjure-undertow-lib
        +--- undertow
```